src/components/transaction/TransactionMessage.js

```diff
@@ -14,17 +14,12 @@ const deviceWidth = deviceUtils.dimensions.width;
 const horizontalPadding = 24;
 
 const Container = styled(Row)({
-  maxHeight: ({ maxHeight }) => maxHeight,
```

Comment:
this file is deprecated and has been moved into the signTransactionSheet

will open a PR post merge to clean up old components